pkg/cmd/utils.go

```diff
@@ -199,3 +200,15 @@ func BuildConfig(master, kubeConfig string, f client.Factory) (*rest.Config, err
 	}
 	return config, nil
 }
+
+func IsFeatureEnabled(featureList string, name string) bool {
```

Comment:
It is unnecessary to implement it on your own. We can use the `features` package from velero. Below is an example.

https://github.com/vmware-tanzu/velero-plugin-for-vsphere/blob/8ab1c4cb6a260fa0f0f681026517f6d83e6398e3/cmd/velero-plugin-for-vsphere/main.go#L34

Reply:
I did try it but could not get it to work. 
I updated the pkg/cmd/backupdriver/backupdriver.go code to add the feature

+               PersistentPreRun: func(cmd *cobra.Command, args []string) {
+                       features.Enable(config.Features()...)
+                       features.Enable(cmdFeatures...)
+               },

+       // Bind features directly to the root command so it's available to all callers.
+       c.PersistentFlags().Var(&cmdFeatures, "features", "Comma-separated list of features to enable for this backup-driver process.")

Still did not see it enabled when i used features.IsEnabled. Maybe I am missing something.

Reply:
```
c.PersistentFlags().AddGoFlagSet(flag.CommandLine)
```
Did you try add the line above afterwards